README.md

```diff
@@ -21,9 +43,11 @@ one of:
 | -------------------------- | -------------------------------------| -----------
 | BIND_ADDR                  | :22900                               | The host and port to bind to
 | CONSUMER_GROUP             | dp-search-builder                    | The name of the Kafka consumer group
-| CONSUMER_TOPIC             | hierarchy-built                      | The name of the Kafka consumer group
-| DATASET_API_URL            | http://localhost:22000               | The host name for the Dataset API
+| CONSUMER_TOPIC             | hierarchy-built                       | The name of the topic to consumes messages from
```

Comment:
This is the same entry as `HIERARCHY_BUILT_TOPIC ` - it seems like the entry for `CONSUMER_TOPIC ` needs removing from the readme